modules/evm/src/lib.rs

```diff
@@ -234,7 +235,9 @@ pub mod module {
 
 	#[derive(Clone, Copy, Eq, PartialEq, RuntimeDebug, Encode, Decode, MaxEncodedLen)]
 	pub struct CodeInfo {
+		#[codec(compact)]
```

Comment:
no need compact for storage related code, it will require a migration and the storage saving on on chain state isn't worth the additional computation